doc/user/content/sql/create-role.md

```diff
@@ -21,27 +21,38 @@ will be used for role-based access control. See GitHub issue {{% gh 677 %}}
 for details.
 {{< /warning >}}
 
+{{< warning >}}
+Attributes currently have no effect.
```

Comment:
Are they persisted and will have an effect once implemented? If so that should be described.

Reply:
Done